src/pages/SignIn/SignIn.scss

```
@@ -1 +1,189 @@
 @import '../../styles/variables.scss';
+@import '../../styles/common.scss';
+
+.signIn {
+  line-height: 1.8rem;
+
+  .title {
+    font-size: 3rem;
+    text-align: center;
+    padding: 3rem;
+    margin-top: 1rem;
+  }
+
+  .containers {
+    display: flex;
+    justify-content: center;
+    // align-items: center;
+    margin-top: 1.5rem;
+    font-size: 1.1rem;
+
+    .borderBottom {
+      width: 1.5rem;
+      border-bottom: 0.25rem solid #cc0000;
+      margin-top: 0.3rem;
+      margin-bottom: 0.3rem;
```

```suggestion
      margin: 0.3rem 0;
```
중복되는 부분은 줄여줄 수 있습니다.

---

수정했습니다!